### PR TITLE
Install template

### PR DESCRIPTION
**Pull request by blu28**
Starting from the code used in week zero, merged in all template code, fixed flagged issues, fixed spelling issues. commented out unused constants and reinstated the anonymous code, but with null returned.

### CODE REVIEW

**Comment by blu28** [comment]
Also updated to WPILIB 2023.4.1

File: src/test/java/frc/robot/TestExample.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package frc.robot;
```
Comment (moso1) [inline src/test/java/frc/robot/TestExample.java:1]: Please delete this unused file.

Comment (moso1) [inline src/test/java/frc/robot/TestExample.java:1]: Please delete this unused file.

**Review by kcq888 — COMMENTED** [review]
I was working on this branch with Vera, and we merged Diya changes into this branch. Everything is tested on the test bench.

File: src/test/java/frc/robot/TestExample.java (hunk 1 above)
Comment (kcq888) [reply src/test/java/frc/robot/TestExample.java:1]: Let's keep this for now

**Pull request merged**